asm/specialized_metadata.go

```diff
@@ -1110,8 +1110,16 @@ func (gen *generator) irDIModule(new metadata.SpecializedNode, old *ast.DIModule
 			md.ConfigMacros = stringLit(oldField.ConfigMacros())
 		case *ast.IncludePathField:
 			md.IncludePath = stringLit(oldField.IncludePath())
-		case *ast.SysrootField:
-			md.Sysroot = stringLit(oldField.Sysroot())
+		case *ast.APINotesField:
+			md.APINotes = stringLit(oldField.APINotes())
+		case *ast.FileField:
+			file, err := gen.irMDField(oldField.File())
+			if err != nil {
+				return nil, errors.WithStack(err)
+			}
+			md.File = file
+		case *ast.LineField:
+			*md.Line = intLit(oldField.Line())
```

Comment:
Would this not result in a nil pointer reference? Since `Line` has not yet been allocated.

Reply:
I'm so surprised deference left value in Go didn't work just like C...